src/extensions/renderer-api/components.ts

```diff
@@ -13,6 +13,9 @@ export * from "../../renderer/components/select";
 export * from "../../renderer/components/slider";
 export * from "../../renderer/components/input/input";
 
+// command-overlay
+export { CommandOverlay } from "../../renderer/components/command-palette";
```

Comment:
`CommandOverlay` or `commandOverlay` (singleton) 🤔 .... naming is hard.

Reply:
`CommandOverlay`, since imo we shouldn't have `commandOverlay` as that basically nullifies our "singleton" code.

Reply:
I meant that should we export `CommandOverlay` (class) or `commandOverlay` (singleton class instance).

Reply:
So looking at `renderer/components/command-palette` I don't even see an instance. Also given that `CommandOverlay` only has static function it won't work at all to export an instance.

Reply:
> So looking at `renderer/components/command-palette` I don't even see an instance. Also given that `CommandOverlay` only has static function it won't work at all to export an instance.

It does not have an instance because I chose to use static functions. I was just wondering should it have 🙂 We probably should use whatever is more consistent with our current extension API.

Reply:
For something like that I think that static function are best